### DISCUSSION TITLE
2008/04/15/%E5%B0%8F%E7%99%BD%E9%BC%A0

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#215] [Announcements]
# 2008/04/15/%E5%B0%8F%E7%99%BD%E9%BC%A0

在没有特殊上下文的时候，小白鼠就是 White Mouse。“小”就不用特别翻译出来了。顺便说一句，我今天搞清楚了 Mouse 和 Rat 的区别。这是两类不同的老鼠，主要外在区别就是体型不一样：Mouse 是小老鼠，Rat 是大老鼠。我这辈子接触的老鼠不多，对它们该有多大，也没概念。所以，估计下次再遇到一只老鼠，恐怕还是不能确定它属于 Mouse 还是 Rat。但是，我自己又总结了一条使用这两个词的规律，那就是：如果是可爱的老鼠，那就要用 Mouse，比如米老鼠、小白鼠；如果令人讨厌的老鼠，比如形容贪官、小偷等的就要用 Rat。

https://blog.qizhen.xyz/2008/04/15/%E5%B0%8F%E7%99%BD%E9%BC%A0

**Comment by ruanqizhen** [comment]
之前博客下的留言：

磊
2008年04月18日 09:11 编辑
生物学讨论?

回复
lynn
2008年04月17日 16:30 编辑
猪，鼠，傻傻分不清楚！

回复
Percy
2008年04月17日 16:06 编辑
你吐个象牙来看看? 吐不出了 换个地你就又是狗狗 哈哈哈~~~

回复
Qizhen
2008年04月17日 15:27 编辑
狗嘴里吐不出象牙；螃蟹嘴里吐出的，顶多也就是几个白泡泡:)

回复
Percy
2008年04月17日 11:39 编辑
哦哦哦~~~就是说换个地你就是鼠，再换个地你就是猪 对伐对伐~~~

回复
Xiaohong
2008年04月17日 10:29 编辑
哼哼，欺负人

回复
Qizhen
2008年04月17日 02:25 编辑
Percy, 这你还理解不了啊。比如哪天天象反常，我老婆居然下厨做饭了，还一定要我先吃。那么，我在中文里就被称为小白鼠；在英文里，被称为荷兰猪。

回复
Nicholas
2008年04月16日 23:10 编辑
不错哈，顶一记～

回复
Percy
2008年04月16日 16:17 编辑
请你再详尽一点阐述“你 小白鼠 荷兰猪”的关系 恩恩~~~

回复
